tests/test_evolutionary_strategies.py

```diff
@@ -0,0 +1,195 @@
+import jax
+from jax import jit, vmap, numpy as jnp
+from flax import linen as nn
+
+from src import evox
+from src.evox import algorithms
+from src.evox.problems.rl.gym_no_distribution import Gym
+from src.evox.utils import TreeAndVector
+
+
+class ClassicPolicy(nn.Module):
+    """A simple model for Classic Control problem"""
+
+    @nn.compact
+    def __call__(self, x):
+        x = nn.Dense(16)(x)
+        x = nn.relu(x)
+        x = nn.Dense(2)(x)
+
+        return jnp.argmax(x)
+
+
+def cartpole(seed, al: type, **kwargs):
```

Comment:
algorithm have type Algorithm, or don't write the type hint.
```suggestion
def cartpole(seed, al, **kwargs):
```